python/nav/web/templates/ipdevinfo/ipdev-details.html

```diff
@@ -47,6 +47,9 @@
       {% try_to_include 'ipdevinfo/custom-header-button-extras.html' %}
 
     </ul>
+
+  {% elif invalid_ip %}
+    <div class='alert-box'>Invalid IP address</div>
```

Comment:
Use double quotes for HTML. Also, use the error class to make a proper error alert box, according to the style guidelines :)

```suggestion
    <div class="alert-box error">Invalid IP address</div>
```